backend/src/user/user.service.ts

```diff
@@ -31,7 +32,7 @@ export class UserService {
       user.email = userInfo.email;
       user.name = userInfo.name;
       user.password = userInfo.password;
-
+      user.bookmarks = ['Implant', 'Condom'];
```

Comment:
Right now, these are hardcoded. Can they be obtained from anywhere? Are they stored in the database?

Reply:
These are hardcoded at the moment because there is no current functionality for adding bookmarks.